community/templates/meetups/attend.html

```diff
@@ -1,4 +1,4 @@
-{% extends "base.html" %}
+{% extends "base_user.html" %}
```

Comment:
should it be base.html? I'm not sure I completely understand which UI templates are for what.  I thought the base_user.html was the unified theme template (w/ dash etc) for a logged in user.  Let me know if I should be using something else.